### PR TITLE
Add Twilio to thanks to and "Supporting People Quarantined or At Risk" track

### PR DESCRIPTION
**Pull request by danielfspencer**

### CODE REVIEW

**Comment by erinosher** [comment]
#69

**Comment by danielfspencer** [comment]
This also adds Twilio to the thanks to section!!!!

**Comment by erinosher** [comment]
image
@danielfspencer can you remove the "Powered by Twilio Asset"

File: _layouts/default.html [hunk 1]
```diff
@@ -39,6 +39,7 @@ <h2>Tracks</h2>
                         <div class="col-sm">
                             <img src="/assets/img/tracks/support.png" width="100%" />
                             <h5>Supporting People Quarantined or At Risk</h5>
+                            <img style="padding-top: 1em; padding-bottom: 1em;" src="/assets/img/tracks/poweredbytwilio.png" width="100%" />
```
Comment (erinosher) [inline _layouts/default.html:42]: This needs to be removed.

**Review by erinosher — CHANGES_REQUESTED** [review]

**Comment by danielfspencer** [comment]
Fix is to Revert #69 and merge #68 and #67

**Pull request closed without merging**